src/renderer/base/renderer.cpp

```diff
@@ -104,6 +104,53 @@ Renderer::~Renderer()
     return S_OK;
 }
 
+template<>
+struct fmt::formatter<TextColor>

```

Comment:
don't need these to be in a generic place, but we could move them into their appropriate headers later